consult.el

```diff
@@ -236,42 +236,56 @@ See `consult--multi' for a description of the source data structure."
   "Regexp used to match file and line of grep output.")
 
 (defcustom consult-grep-args
-  "grep --null --line-buffered --color=never --ignore-case --line-number -I -r ."
+  '("grep" (consult--grep-generate-exclude-args)
+    "--null --line-buffered --color=never\
+    --ignore-case --line-number -I -r .")
   "Command line arguments for grep, see `consult-grep'.
-The dynamically computed arguments are appended."
-  :type 'string)
+The dynamically computed arguments are appended.
+
+Can be either a string, or a list of string or expressions."
+  :type '(choice string (repeat string)))
```

Comment:
Sorry, what I mentioned about the type before was wrong. Should be (choice string (repeat (choice string expression))).